db/Controllers/reviews.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchAllReviews } = require("../Models/reviews.models")
+
+exports.getAllReviews = (req,res, next) => {
+    const {reviewID} = req.params
+    fetchAllReviews(reviewID).then((reviews)=>{
+        res.status(200).send({reviews})
```

Comment:
You're sending a single review, so this should be on the `review` key rather than `reviews`
(this will impact your tests sorry!)